crates/mipsy_interactive/src/interactive/mod.rs

```diff
@@ -480,8 +482,9 @@ fn editor() -> Editor<MyHelper> {
     let helper = MyHelper::new();
     rl.set_helper(Some(helper));
 
-    rl.bind_sequence(KeyPress::ControlLeft,  Cmd::Move(Movement::BackwardWord(1, Word::Emacs)));
-    rl.bind_sequence(KeyPress::ControlRight, Cmd::Move(Movement::ForwardWord (1, At::BeforeEnd, Word::Emacs)));
+    use {KeyCode as K, KeyEvent as E, Modifiers as M};
```

Comment:
nit: I think I'd prefer this as `use {KeyCode::{Left, Right}, Modifers::CTRL};` and simply using the whole `KeyEvent` variant name.